.circleci/config.yml

```diff
@@ -16,6 +16,10 @@ defaults:
 
       cp .circleci/prepare-commit-msg .git/hooks/prepare-commit-msg
       chmod +x .git/hooks/prepare-commit-msg
+  persist_repo: &persist_repo
+    root: ~/
+    paths:
+      - project
```

Comment:
To confirm: we're now persisting the *all* of the project content (including the transpiled JavaScript artifacts, modules, etc.)?

Reply:
Yes, anything that's under the repo folder: source code, transpiled artifacts, `node_modules`, and the `.git` directory.

Reply:
Got it, thanks.